components/prism-tremor.js

```diff
@@ -0,0 +1,39 @@
+// NOTE: Tremor supports recursive nested interpolation in strings
+// but we do support intperpolation in this definition as nested
+// recursive interpolation isn't yet fully supported by prism.
+
+Prism.languages.tremor = {
+	'comment': {
+		pattern: /(^|[^\\])(?:\/\*[\s\S]*?\*\/|(?:--|\/\/|#).*)/,
+		lookbehind: true
+	},
+   	'heredoc': {
+        pattern: /"""(?:[^"]|("[^"])|(""[^"]))+"""/,
+        greedy: true,
+        alias: "string"
+	},
+   	'string': {
+        pattern: /"(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])+"|""/,
```

Comment:
This brings the worst-case time complexity down from O(2^n) to O(n).

```suggestion
        pattern: /(^|[^\\])"(?:\\(?:\r\n|[\s\S])|[^"\\\r\n])*"/,
        lookbehind: true
```